modules/openthread/platform/crypto_psa.c

```diff
@@ -134,13 +136,41 @@ otError otPlatCryptoImportKey(otCryptoKeyRef *aKeyRef,
 			      const uint8_t *aKey,
 			      size_t aKeyLen)
 {
+	int version;
+    size_t len;
+    unsigned char *p = (unsigned char *) aKey;
 	psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
-	psa_status_t status;
+	psa_status_t status = 0;
+	unsigned char *end;
 
 	if (aKeyRef == NULL || aKey == NULL || !checkKeyUsage(aKeyUsage)) {
 		return OT_ERROR_INVALID_ARGS;
 	}
 
+	/* Check if key is ECDSA pair and extract private key from it since PSA expects it. */
+	if (aKeyType == OT_CRYPTO_KEY_TYPE_ECDSA) {
```

Comment:
should we guard these parts with `#if defined(CONFIG_OPENTHREAD_ECDSA)` ?

Reply:
fixed.